src/main/java/com/google/devtools/build/lib/actions/FileContentsProxy.java

```diff
@@ -21,38 +21,60 @@
 
 /**
  * In case we can't get a fast digest from the filesystem, we store this metadata as a proxy to the
- * file contents. Currently it is a pair of a relevant timestamp and a "node id". On Linux the
- * former is the ctime and the latter is the inode number. We might want to add the device number in
+ * file contents. Currently it is two timestamps and a "node id". On Linux we
+ * use both ctime and mtime and inode number. We might want to add the device number in
  * the future.
  *
- * <p>For a Linux example of why mtime alone is insufficient, note that 'mv' preserves timestamps.
+ * <p>For a Linux example of ctime alone is insufficient, and we add node id,
```

Comment:
This example is no longer correct. `mv` does change `ctime`. Not sure then if `inode` is required, though.

Reply:
You are right.
I think this example is confusing to me in the way it is written.
Since mtime was not present in the class, but referenced in the documentation,
I assumed they meant ctime here, and that they added nodeId as a complement.
I will revert this change in the documentation. But I still don't know what it is trying to explain, I guess then what it tries to tell me is why ctime is preferred over mtime?